src/mimic.coffee

```diff
@@ -46,6 +46,13 @@ module.exports = (robot) ->
 
   models = {}
 
+  getModel = (user_name) ->
+    return all_model if user_name == 'all'
+    user = robot.brain.userForName(user_name)
+    return null if !user
+    models[user.id] = models[user.id] or makeNewModel(user.id)
```

Comment:
I've set a bad example below (line 62) with this pattern, since I believe `models[user.id] ||= makeNewModel(user.id)` is more idiomatic CoffeeScript. I appreciate your keeping the file's style consistent, but it would be cool to improve it in this case.